corehq/apps/userreports/dbaccessors.py

```diff
@@ -0,0 +1,10 @@
+def get_report_configs_by_data_source(domain, data_source_id):
```

Comment:
@czue in my own refactors, I've been putting things in one big `dbaccessors.py` file per app. My reasoning is that it emphasizes that it doesn't matter what kind of backend you're using, and encourages rewriting to use a different view if that means deleting/consolidating views. The other convention doesn't totally prevent that, just means if you rewrite a function's internals you also have to move it to another file.